_drafts/on_testing_degeneracy_of_games.md

```diff
@@ -193,17 +193,35 @@ is sufficient to test.
 
 ## A sufficient mixed strategy to test for degeneracy
 
-Let \\(X\\) be the set of all strategies for Player1.
-Let \\(Y\\) be the set of all strategies for Player2.
+Given a support \\(\sigma\\) for Player 1, we want to find a strategy \\(x\\)
+with support \\(\sigma\\) that maximises the number of best respones for Player 2.
+If the game is degenerate then there will exist an \\(x\\) which has more best
+responses than the size of its support \\(\sigma\\).
+
+Here we show that for a degenerate game, it is sufficient to make Player 1 and
```

Comment:
This isn't quite what we want, they don't need to be indifferent to each other. We just need that for x makes player two indifferent for each size of support for player 2 and then subsequently (the algorithm might not even get here) that for y it makes player 1 indfferent for each of their supports.